Kotlin/sort/Shell Sort/src/ShellSort.kt

```diff
@@ -0,0 +1,59 @@
+// To Implement Shell Sort in Kotlin
```

Comment:
Add a small description of shell sort algorithm.

Reply:
Added